var/httpd/htdocs/skins/Agent/default/css/Core.Agent.TicketProcess.css

```diff
@@ -465,6 +465,10 @@ did not receive this file, see https://www.gnu.org/licenses/gpl-3.0.txt.
         color: var(--gray-dark-semi);
     }
 
+    .NoMarginTop {
+        margin-top: 0px !important;
+    }
+
```

Comment:
```suggestion
```

Not needed: [var/httpd/htdocs/skins/Agent/default/css/Core.FlexboxModel.css](https://github.com/znuny/Znuny/blob/dev/var/httpd/htdocs/skins/Agent/default/css/Core.FlexboxModel.css#L163)